Runtime/RenderPipeline/PSXRenderPipeline.cs

```diff
@@ -906,6 +931,9 @@ void PushGlobalRasterizationParameters(Camera camera, CommandBuffer cmd, RTHandl
                 
                 float time = GetAnimatedMaterialsTime(camera);
                 cmd.SetGlobalVector(PSXShaderIDs._Time, new Vector4(time / 20.0f, time, time * 2.0f, time * 3.0f));
+                
+                float timeUnscaled = GetAnimatedMaterialsTimeUnscaled(camera);
+                cmd.SetGlobalVector(PSXShaderIDs._TimeUnscaled, new Vector4(timeUnscaled / 20.0f, timeUnscaled, timeUnscaled * 2.0f, timeUnscaled * 3.0f));
```

Comment:
While you are at it - can you add this same time unscaled option to the CRT shader? It's been requested before, and you are touching the same shader.

Reply:
It should be pretty simple to round the time variable used in the panning of the noise such that there's no noticeable difference at full FPS but would animate frame-by-frame based on the time scale relative to the FPS of the game.

Reply:
Just fixed this, by stepping the timescale being used to every 1/60th second, so lower timescales don't show the scrolling texture